scripts/test/set-e2e-env-vars.sh

```diff
@@ -19,6 +19,13 @@ fi
 # Set environment variables
 export AWS_WORKSPACE="${WORKSPACE}"
 export MTLS_ENDPOINT="mtls.${WORKSPACE}.${DOMAIN}"
+if [[ "$WORKSPACE" = "pre-prod" ]]; then
```

Comment:
yeah same, so can we not just move this into the DataHelper class and use the workspace env variable?